src/hijackers/dynamicScript.ts

```diff
@@ -0,0 +1,26 @@
+import { execScripts } from 'import-html-entry';
+import { Freer } from '../interfaces';
+
+export default function hijack(proxy: Window): Freer {
+  const rawHtmlAppendChild = HTMLHeadElement.prototype.appendChild;
```

Comment:
move to the top of module to avoid using the polluted value, ref https://github.com/umijs/qiankun/blob/master/src/hijackers/dynamicStylesheet.ts#L8

Reply:
if the function move to the top, it will override the dynamicStylesheet function hijack

Reply:
we may need to combine the two into one hijacker, as they are overwrite the same native function. rename the `dynamicStylesheet` to `dynamicHeadAppend` and move the script injection logic into it?

Reply:
i updated